ui-manchette-with-spacetimechart/src/helpers.ts

```diff
@@ -154,17 +159,32 @@ export const getScales = (
   ];
 };
 
+export const zoomValueToTimeScale = (slider: number) =>
+  MIN_ZOOM_MS_PER_PX * Math.pow(MAX_ZOOM_MS_PER_PX / MIN_ZOOM_MS_PER_PX, slider / 100);
+
+export const timeScaleToZoomValue = (timeScale: number) =>
+  (100 * Math.log(timeScale / MIN_ZOOM_MS_PER_PX)) /
+  Math.log(MAX_ZOOM_MS_PER_PX / MIN_ZOOM_MS_PER_PX);
+
 /** Zoom on X axis and center on the mouse position */
 export const zoomX = (
   currentXZoom: number,
   currentXOffset: number,
-  { delta, position: { x } }: Parameters<NonNullable<SpaceTimeChartProps['onZoom']>>[0]
+  newXZoom: number,
```

Comment:
Nit: since this function is named `zoomX`, it's probably fine to drop the `X` here.

Reply:
done, dropped all the xs in the function